pkg/virt-controller/watch/vmi/vmi_test.go

```diff
@@ -767,6 +769,44 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 		})
 	})
 
+	It("should succeed which when network VMI spec validator succeed", func() {
+		controller.validateNetworkSpec = validateNetVMISpecStub()
+
+		vmi := newPendingVirtualMachine("testvmi")
+		addVirtualMachine(vmi)
+
+		sanityExecute()
+
+		testutils.ExpectEvent(recorder, kvcontroller.SuccessfulCreatePodReason)
+
+		vmi, err := virtClientset.KubevirtV1().VirtualMachineInstances(vmi.Namespace).Get(context.Background(), vmi.Name, metav1.GetOptions{})
+		Expect(err).ToNot(HaveOccurred())
+		Expect(vmi.Status.Conditions).To(HaveLen(1))
+		Expect(vmi.Status.Conditions).To(SatisfyAll(ContainElement(MatchFields(IgnoreExtras, Fields{
+			"Type":   Equal(virtv1.VirtualMachineInstanceReady),
+			"Status": Equal(k8sv1.ConditionFalse),
+			"Reason": Equal(virtv1.GuestNotRunningReason),
+		}))))
+	})
+	It("should fail which when network VMI spec validator fail", func() {
+		controller.validateNetworkSpec = validateNetVMISpecStub(metav1.StatusCause{Type: "test", Message: "test", Field: "test"})
+
+		vmi := newPendingVirtualMachine("testvmi")
+		addVirtualMachine(vmi)
+
+		sanityExecute()
+
+		vmi, err := virtClientset.KubevirtV1().VirtualMachineInstances(vmi.Namespace).Get(context.Background(), vmi.Name, metav1.GetOptions{})
+		Expect(err).ToNot(HaveOccurred())
+		Expect(vmi.Status.Conditions).To(HaveLen(2))
```

Comment:
What is the second condition being asserted here?
Are we interested in it?

Reply:
Not really, looking at this again this assertion make no sense, I removed it.